src/client/containers/EventsFakeAPI.js

```diff
@@ -0,0 +1,103 @@
+// All properties are arrays of events
+// const EventsResponse = {
+//     "TODAY": [],
+//     "TOMORROW": [],
+//     "WEEK": [],
+//     "MONTH": []
+// }
+// Fake Api
+
+// Events Info for PageInfoComponent. Notice that it misses the color
+export const EventsPageInfo = {
+  // color: '#253F51',
+  title: 'EVENTS',
+  info: `Many events are organized throughout the semester. 
+    From companies' info sessions to simple workshops, 
+    we offer you many opportunities to become successful in your studies`,
+};
+
+// Might be moved to API where the weekdays are defined
+export const ProvidedTimes = [
+  'TODAY',
+  'TOMORROW',
+  'THIS WEEK',
+  'THIS MONTH',
+]
+
+export const EventField = [
+  'month',
+  'day',
+  'weekday',
+  'name',
+  'timeloc',
+  'content',
+]
+
+
+export const EventsResponse = {
```

Comment:
This structure is not very realistic as to how an API is going to give you data. Think about this. What if we tried to take out the groupings of events (remove 'TODAY', 'TOMORROW', etc.). We would have to change the way the API sends data. If you handed this kind of string building on the client, given a start and end date and location, we would only change the client and not the API.